frameworks/kafka/src/test/java/com/mesosphere/sdk/kafka/scheduler/ServiceTest.java

```diff
@@ -52,6 +52,6 @@ private ServiceTestRunner getDefaultRunner() {
 
         return new ServiceTestRunner()
                 .setPodEnv("kafka", map)
-                .setBuildTemplateParams("kafka-version", "2.11-1.0.0"); // set by build.sh/versions.sh
+                .setBuildTemplateParams("kafka-version", "2.12-1.1.0"); // set by build.sh/versions.sh
```

Comment:
We could use a dummy version number here so that we don't need to keep updating it for every version bump. This was also done for elastic in #2536.

Reply:
sounds good to me

Reply:
fixed in commit [1f9fe95](https://github.com/mesosphere/dcos-commons/pull/2542/commits/1f9fe955e4ed6f60b4b03f56bb80313341b3dbce)